sectionsUtilityTool/README.txt

```diff
@@ -13,6 +13,8 @@
 canvas.admin.token=
 # eg.https://umich.test.instructure.com
 canvas.url=
+# when cosign/local development is not enable set it true. Production should always be set it to false. We are enabling testUser for local develpemt and with this variable testUser will never is enabled. 
+test.url=false
```

Comment:
Change name to something like "use.test.url".  From the name I think most people would expect the value to actually be an URL.